src/main/java/org/folio/circulation/domain/UpdateItem.java

```diff
@@ -208,9 +210,13 @@ private ItemStatus itemStatusOnLoanUpdate(
     Loan loan,
     RequestQueue requestQueue) {
 
-    return loan.isClosed()
-      ? itemStatusOnCheckIn(requestQueue)
-      : CHECKED_OUT;
+    if(loan.isClosed()) {
+      return itemStatusOnCheckIn(requestQueue);
+    }
+    if(Objects.equals(loan.getItem().getStatus(), ItemStatus.DECLARED_LOST)) {
+      return DECLARED_LOST;
+    }
+    return  CHECKED_OUT;
```

Comment:
What happens if this is changed to use the item status as it is set e.g. `return loan.getItem().getStatus()`?

Reply:
@marcjohnson-kint I think that it should be a special status because we have 3 other values, returning from this method. In the previous implementation, there were only 2 ('CHECKED_OUT' or from 'itemStatusOnCheckIn' method). So, if changed to returning only the item status, when it not as 'CLOSED' then we lose 'CHECKED_OUT' value from this method. 

Or did you mean just change `return DECLARED_LOST;` on `return loan.getItem().getStatus()` ?


Did I understand you correctly?

Reply:
Yes, I meant that rather than returning either `Checked out` or `Declared lost` specifically, instead, use the status from the item itself (which it should probably be all of the time, and the other special case should also go away)

Reply:
@marcjohnson-kint Yeah, but I want to mention that in the previous implementation there was hardcoded returning CHECKED_OUT status when the loan isn`t closed. So, that`s why the right way to do it - it is only returning status from item only in case with DECLARED_LOST. Does it make sense?

Reply:
I don't think I follow that explanation. Why is the declared lost status special?

I'm suggesting something similar to the following:
```
if(loan.isClosed()) {
      return itemStatusOnCheckIn(requestQueue);
    }
    else {
      return loan.getItem().getStatus();
    }
```

Reply:
@marcjohnson-kint Ok. I can do this. But I worried that this approach can cause some problems in the system because CHECKED_OUT was hardcoded. But if you think that everything will be ok, I will do it.

Reply:
I think it is worth an experiment

Reply:
@marcjohnson-kint  So, with this implementation, we have failed test: canCreateALoanAtASpecificLocation().

That`s why I think that the first way is better.

Reply:
@VRohach thats cool, if it breaks tests, let’s use the conservative approach you did. I only wanted to try it.

Reply:
@marcjohnson-kint Sure! Thank you for your support!